docs/index.rst

```diff
@@ -12,10 +15,17 @@ designed to facilitate packaging Python projects.
    build_meta
    pkg_resources
    references/keywords
-   roadmap
    setuptools
+
+.. toctree::
+   :caption: Project
+   :maxdepth: 1
+   :hidden:
+
+   roadmap
    Development guide <development/index>
    Backward compatibility & deprecated practice <deprecated/index>
    Changelog <history>
+   Artwork <images/README>
```

Comment:
I don't know exactly where to place it, so I added it here, but no strong preference...